common/wuanalysis/anacommon.hpp

```diff
@@ -44,4 +46,11 @@ class PerformanceIssue : public CInterface
 
 extern int compareIssuesCostOrder(CInterface * const * _l, CInterface * const * _r);
 
+struct WuAnalyseOptions
```

Comment:
This was original intended to be a private include file - this should be in anawu.hpp which is the public interface to the analyser

Reply:
I understand. Will move public interface to anawu.hpp.